### DISCUSSION TITLE
Running PowerDNS on K8s

### DISCUSSION THREAD

**Discussion by kurthv** [PowerDNS/pdns#13974] [Q&A]
I am looking for advice on how to run PowerDNS Authoritative Server on Kubernetes.
I already found a Helm chart (https://github.com/cdwv/powerdns-helm), but it's not updated for approx. 3 years using pdns 4.2.0.

My questions:
1. Is there a Helm Chart available for recent relaeses (e.g. v4.9.x)?
2. If not, can somebody provide me some information/documentation on how to deploy PowerDNS on Kubernetes using MariaDB as database?

Thank you very much in advance!

Kind regards,
Volker

**Comment by nakroy** [comment]
I guess you're looking for this:
https://www.powerdns.com/powerdns-cloud-control
But it seems that this is an enterprise product

**Comment by owndomainhome** [comment]
@kurthv , you can use this: https://github.com/owndomainhome/powerdns-stack-over-k8s.

**Reply by arthurkay** [reply]
The documentation is not clear on how to bootstrap the AS sqlite db

**Reply by eporro** [reply]
@arthurkay Sure!, "how to bootstrap of the AS role sqlite db" is not the main goal of the resource, PowerDNS is, and the way of deployment over kubernetes.
